README.md

```diff
@@ -0,0 +1,10 @@
+## jni_gen
+
+This project intends to provide
```

Comment:
Maybe it would be cool to provide a table as in https://github.com/google/protobuf.dart.